giraph-core/src/main/java/org/apache/giraph/conf/GiraphConstants.java

```diff
@@ -1201,6 +1203,13 @@
       new BooleanConfOption("giraph.trackJobProgressOnClient", false,
           "Whether to track job progress on client or not");
 
+  /** Class to use as the job progress client */
+  ClassConfOption<JobProgressTrackerClient> JOB_PROGRESS_TRACKER_CLIENT_CLASS =
+      ClassConfOption.create("giraph.jobProgressTrackerClientClass",
+        RetryableJobProgressTrackerClient.class,
+          JobProgressTrackerClient.class,
+          "Class to use to make calls to the job progress tracker service");
+
   /** Class to use to track job progress on client */
   ClassConfOption<JobProgressTrackerService> JOB_PROGRESS_TRACKER_CLASS =
```

Comment:
As you are adding the JobProgressTrackerClient, can we rename this to JobProgressTrackerService, to be clear about the difference between the two?